charts/openebs/templates/deployment-local-provisioner.yaml

```diff
@@ -70,6 +70,12 @@ spec:
           value: "{{ .Values.image.repository }}{{ .Values.helper.image }}:{{ .Values.helper.imageTag }}"
         - name: OPENEBS_IO_INSTALLER_TYPE
           value: "charts-helm"
+        # LEADER_ELECTION_ENABLED is used to enable/disable leader election. By default
+        # leader election is enabled. If spec.replicas equal to 1 then disable leader election
+{{- if eq .Values.localprovisioner.replicas 1.0 }}
+        - name: LEADER_ELECTION_ENABLED
+          value: "false"
```

Comment:
@kmova JFYI we are disabling the leader election in case of single replica.

Reply:
Or we should leave it to user to decide ?

Reply:
I think we are using different options depending on operator yaml vs helm. Let us keep it the same. It will be good to add a new parameter in values and based on that set the value.